docs/configuring-metabase/environment-variables.md

```diff
@@ -185,6 +185,14 @@ Since: v35.0
 
 Maximum number of async Jetty threads. If not set, then [MB_JETTY_MAXTHREADS](#mb_jetty_maxthreads) will be used, otherwise it will use the default.
 
+### `MB_ATTACHMENT_TABLE_ROW_LIMIT`
+
+Type: integer<br>
+Default: `20`<br>
+Since: v48.0
```

Comment:
> Since: v48.0

Has this env var been in since v48.0? 

In any case, we've only documented version numbers on some but not all env vars, and I don't think it's necessary to include this info as our docs are already versioned.

Reply:
Sorry, 48.2.

Ok! Saw it for some of them and added it because of that.